modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/liferay-frontend.tld

```diff
@@ -335,6 +335,30 @@
 		</attribute>
 	</tag>
 	<tag>
+		<description><![CDATA[Creates a UI component for managing icon.]]></description>
+		<name>icon</name>
+		<tag-class>com.liferay.frontend.taglib.servlet.taglib.soy.IconTag</tag-class>
+		<body-content>empty</body-content>
+		<attribute>
+			<description><![CDATA[CSS classes for styling the icon container and also define icons that was not defined as SVG.]]></description>
+			<name>elementClasses</name>
```

Comment:
@brunobasto, I would like to highlight this attribute name here `elementClasses`. I noticed that taglibs have used `cssClass` for this attribute type. My idea of using `elementClass` is to maintain consistency between MetalJS and soy taglibs. On the other hand, I know that soy taglibs should have as little impact as possible on the migration process. I'm not sure if this kind of change, even small, worth. What do you think?